capsules/extra/src/kv_store.rs

```diff
@@ -191,6 +242,10 @@ impl<'a, K: KVSystem<'a, K = T>, T: kv_system::KeyType> KVStore<'a, K, T> {
         self.mux_kv.do_next_op();
         Ok(())
     }
+
+    pub fn header_size(&self) -> usize {
+        mem::size_of::<KeyHeader>()
+    }
```

Comment:
This seems redundant with the `HEADER_LENGTH` const at the top of the file

Reply:
Yeah, result of different changes at different times. I can change this to `HEADER_LENGTH`.